browser/ai_chat/common/BUILD.gn

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+source_set("common") {
```

Comment:
this should be in `components`.

Reply:
only if it's needed to be used in components, right @goodov. Otherwise best practice seems to be keep in `brave/browser/[feature]` (without the `common` subdirectory of course).